recipes/napari-laptrack/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "napari-laptrack" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/napari-laptrack-{{ version }}.tar.gz
+  sha256: e92406c812e601fcb04e030d995c625e58a2d64fe8b02fed565426caab346b26
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - setuptools-scm
+  run:
+    - laptrack >=0.10.0
+    - napari-plugin-engine >=0.1.4
+    - napari-time-slicer
+    - napari-tools-menu
+    - napari-skimage-regionprops >=0.9.0
+    - numpy
+    - pandas
+    - python >=3.6
+
+test:
+  imports:
+    - napari_laptrack
+  commands:
+    - pip list
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/haesleinhuepf/napari_laptrack
```

This URL 404's, should be:
```suggestion
  home: https://github.com/haesleinhuepf/napari-laptrack
```